src/main.ts

```diff
@@ -56,12 +58,25 @@ export function run(
   downloadUrl: string = DOWNLOAD_URL,
   executable: string = EXECUTABLE,
   coverageCommand: string = DEFAULT_COVERAGE_COMMAND,
+  workingDirectory: string = DEFAULT_WORKING_DIRECTORY,
   codeClimateDebug: string = DEFAULT_CODECLIMATE_DEBUG,
   coverageLocationsParam: string = DEFAULT_COVERAGE_LOCATIONS,
   coveragePrefix?: string
 ): Promise<void> {
   return new Promise(async (resolve, reject) => {
     let lastExitCode = 1;
+    if (workingDirectory) {
```

Comment:
```suggestion
    if (workingDirectory.length) {
```

This way, this `if` block doesn't have to be executed if the working directory is the default.